Eastern-Research-Group/expert-query
# Feature/83 add configuration eq service url

## Related Issues:
* [EQ-83](https://jira.epa.gov/browse/EQ-83)

## Main Changes:
* Updated the API Query and curl boxes to be tied to the eqDataApi variable for when we switch to using api.epa.gov.

## Steps To Test:
1. Navigate to http://localhost:3000/attains#dataProfile=assessmentUnits&format=csv&state=AL&state=AS
2. Verify the `Assessment Units API Query` and `cURL` boxes have the `http://localhost:9090/attains/data/assessmentUnits` url 
3. In the `app\server\app\content\config\services.json` file, change the `eqDataApi` value to `https://api.epa.gov/expert-query/attains`
4. Navigate to http://localhost:3000/attains#dataProfile=assessmentUnits&format=csv&state=AL&state=AS
5. Verify the `Assessment Units API Query` and `cURL` boxes have the `https://api.epa.gov/expert-query/attains/data/assessmentUnits` url
